File: palindrome.java
package Maths;

public class palindrome {
    static boolean palindrome(int n)
    {
        int a=0;
        int r=n%10;
        a=a+10*r;
        n=n/10;
        if(a==n)
        {
            return true;
        }
        else {
            return false;
        }
    }
    public static void main(String[] args) {
        System.out.println(palindrome(370));

    }
}

File: palindrome.py
def palindrome(num):
    '''Function to find out the palindrome of a number'''
    temp = num
    rev = 0
    while temp > 0:
        dig = temp % 10
        rev = rev * 10 + dig
        temp = temp // 10
    if num == rev:
        print("The number is a palindrome!")
    else:
        print("The number isn't a palindrome!")
